include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -44,10 +44,8 @@ enum { COL_NAME, COL_ARCH, COL_EVR, COL_REPO, COL_SIZE };
 static const char * action_color(libdnf::transaction::TransactionItemAction action) {
     switch (action) {
         case libdnf::transaction::TransactionItemAction::INSTALL:
-            return "green";
```

Comment:
Please make a commit message shorter. What about to remove ` libdnf/transaction:`?

The prefix makes duplicity of transaction word.